locale/pt-br/texts.po

```diff
@@ -0,0 +1,1422 @@
+"Mensagem de Erro: ${ message }\n"
+"Stack trace: ${ stack }"
+
+#: src/components/ModalUnhandledError.js:67
+msgid "Ooops... Something went wrong..."
+msgstr "Ooops... Algo deu errado..."
+
+#: src/components/ModalUnhandledError.js:70
+msgid "An unhandled exception occurred in your wallet."
+msgstr "Ocorreu um erro não esperado na sua wallet."
+
+#: src/components/ModalUnhandledError.js:71
+msgid ""
+"If you'd like to help us improve it, please click in the button bellow to "
+"copy the error details to your clipboard."
+msgstr ""
+"Se você puder nos ajudar a melhorar a wallet, por favor clique no botão "
+"abaixo para copiar os detalhes do erro."
+
+#: src/components/ModalUnhandledError.js:75
+msgid "Copy Error"
+msgstr "Copiar Erro"
+
+#: src/components/ModalUnhandledError.js:78
+msgid "Send error report to help the developers"
+msgstr "Enviar o relatório de erros para ajudar os desenvolvedores"
+
+#: src/components/ModalUnhandledError.js:80
+msgid ""
+"Send it to us with all relevant information about the issues you are "
+"experiencing."
+msgstr ""
+"Envie para nós todas as informações relevantes sobre o problema que está "
+"ocorrendo."
+
+#: src/components/ModalUnhandledError.js:81
+msgid ""
+"If you are facing recurrent issues, resetting your wallet may solve your "
+"problem. Please, double-check your backup before doing it."
+msgstr ""
+"Se você está tendo problemas recorrentes, você pode resetar sua wallet para "
+"ver se resolve. Por favor, confira o seu backup antes de resetá-la."
+
+#: src/components/ModalUnhandledError.js:88
+msgid "Home"
+msgstr "Início"
+
+#: src/components/ModalUnhandledError.js:89
+msgid "Dismiss"
+msgstr "Descartar"
+
+#: src/components/ModalUnhandledError.js:90
+msgid "Reset wallet"
+msgstr "Resetar wallet"
+
+#: src/components/ModalUnregisteredTokenInfo.js:81
+msgid "Unregistered token"
+msgstr "Token desregistrado"
+
+#: src/components/ModalUnregisteredTokenInfo.js:97
+msgid ""
+"This token is <strong>not registered</strong> in your wallet. You must "
+"<strong>always validate the token uid</strong>, to ensure you are not being "
+"scammed."
+msgstr ""
+"Esse token <strong>não está registrado</strong> na sua wallet. Você deve "
+"<strong>sempre validar o uid to token</strong>, para ter certeza que não "
+"está sendo enganado."
+
+#: src/components/ModalUnregisteredTokenInfo.js:98
+msgid "The token uid is always unique, and your only trust point."
+msgstr ""
+"O uid do token é sempre único e a única informação que você deve confiar."
+
+#: src/components/ModalUnregisteredTokenInfo.js:103
+msgid "I want to register this token"
+msgstr "Eu quero registrar esse token"
+
+#: src/components/ModalUnregisteredTokenInfo.js:111
+msgid "Register token"
+msgstr "Registrar um token"
+
+#: src/components/Navigation.js:37
+msgid "Wallet"
+msgstr "Wallet"
+
+#: src/components/Navigation.js:40
+msgid "Send tokens"
+msgstr "Enviar tokens"
+
+#: src/components/Navigation.js:43
+msgid "Custom tokens"
+msgstr "Tokens"
```

Comment:
Temos que discutir se vale a pena continuar usando 'Custom tokens' em inglês.

Reply:
I agree. Maybe we should change to "Tokens".